types/swagger-stats/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "private": true,
     "dependencies": {
-        "fastify": ">=2.11.0",
+        "fastify": "^3.0.0",
```

Comment:
Pin Fastify < 4.0.0 for now. 4.0.0 was released yesterday and the TypeScript compiler fails:
```
error TS2344: Type 'SchemaCompiler' does not satisfy the constraint 'FastifySchema'.
```

Apart from a glance at the Fastify issue tracker I haven't investigated.

swagger-stats depends on Fastify and `@types/node`, so `@types/node` edits trigger the swagger-stats tests, which fail without this change.